packages/@okta/vuepress-site/docs/reference/registration-hook/index.md

```diff
@@ -85,16 +120,19 @@ The following commands are supported for the Registration Inline Hook type:
 
 | Command                      | Description                                                  |
 |------------------------------|--------------------------------------------------------------|
-| com.okta.user.profile.update | Change values of attributes in the user's Okta user profile. |
+| com.okta.user.profile.update | Change values of attributes in the user's Okta user profile. For SSR only. Invalid if used with a Progressive Profile response.  |
 | com.okta.action.update       | Allow or deny the user's registration.                       |
+| com.okta.user.progressive.profile.update   | Change values of attributes in the user's Okta Progressive Profile (Okta Identity Engine only). |
 
 To set attributes in the user's Okta profile, supply a type property set to `com.okta.user.profile.update`, together with a `value` property set to a list of key-value pairs corresponding to the Okta user profile attributes you want to set. The attributes must already exist in your user profile schema.
 
 To explicitly allow or deny registration to the user, supply a type property set to `com.okta.action.update`, together with a value property set to `{"registration": "ALLOW"}` or `{"registration": "DENY"}`. The default is to allow registration.
 
-Commands are applied in the order in which they appear in the array. Within a single `com.okta.user.profile.update` command, attributes are updated in the order in which they appear in the `value` object.
+In Okta Identity Engine, to set attributes in the user's Okta progressive profile, supply a type property set to `com.okta.user.progressive.profile.update`, together with a `value` property set to a list of key-value pairs corresponding to the progressive profile  attributes you want to set. See [Registration Inline Hook - Send response](/docs/guides/registration-inline-hook/nodejs/main/#send-response) for more information. <ApiLifecycle access="ie" />
```

Comment:
Should we capitalize "Okta progressive profile"? It's capitalized in the previous section.
Remove the extra space after `progressive profile  attributes`